### PR TITLE
fix(project options): Store feedback:branding options correctly

### PR DESCRIPTION
**Pull request by ceorourke**
Toggling off the user feedback branding option wasn't actually doing anything - when the user refreshed the screen it was back to "on". This PR fixes that by grabbing the option correctly - before it was always falling back to the default value and setting it to True.

### CODE REVIEW

**Comment by ceorourke** [comment]
It looks like this change was made recently in https://github.com/getsentry/sentry/pull/39575 so cc @cathyteng17 and @markstory

**Review by wedamija — APPROVED** [review]

**Review by cathteng — APPROVED** [review]
thanks for catching this!

File: src/sentry/api/serializers/models/project.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def format_options(attrs: defaultdict(dict)):
         f"filters:{FilterTypes.ERROR_MESSAGES}": "\n".join(
             attrs["options"].get(f"sentry:{FilterTypes.ERROR_MESSAGES}", [])
         ),
-        "feedback:branding": attrs.get("feedback:branding", "1") == "1",
+        "feedback:branding": attrs["options"].get("feedback:branding", "1") == "1",
```
Comment (wedamija) [inline src/sentry/api/serializers/models/project.py:187]: I can't comment on it, but is `"sentry:reprocessing_active": bool(attrs.get("sentry:reprocessing_active", False)),` also broken?

Comment (wedamija) [reply src/sentry/api/serializers/models/project.py:187]: Maybe we should do
```
options = attrs["options"]
....
    options.get(...)
```

Might be less error prone

Comment (cathyteng17) [reply src/sentry/api/serializers/models/project.py:187]: i think you're right (on both counts)

**Review by wedamija — APPROVED** [review]

**Review by dashed — APPROVED** [review]

**Review by markstory — APPROVED** [review]

**Pull request merged**